portal/code/cx-portal/.env

```diff
@@ -1,2 +1,4 @@
 REACT_APP_BASE_CENTRAL_IDP=https://catenaxdev003akssrv.germanywestcentral.cloudapp.azure.com/iamcentralidp/auth
 REACT_APP_BASE_UTIL=https://catenax-dev003-util.azurewebsites.net/v1
+REACT_APP_BPDM_API_BASE_URL=https://catenax-bpdm-dev.germanywestcentral.cloudapp.azure.com/api
```

Comment:
I want to work on Error handling may be in another PR because it looks like quite a lot to do there. I agree it's need definitely needs more proper error handling in api calls. For timeout issue, I also get timeout. Most probably that api is down for a moment. I will inform my responsible colleague to fix it. Swagger ui url : https://catenax-bpdm-dev.germanywestcentral.cloudapp.azure.com/api/swagger-ui

Reply:
We should not expose buggy features to the test users because this leaves a bad impression. So change the access to this page from EVERYONE to some role that only we developers or people who want to test that feature can see. So in Keycloak create a role portal_developer or partner_network_developer that you assign to yourself: 
`AccessService.ts` line 17
`[PAGES.PARTNER_NETWORK]: ROLES.PORTAL_DEVELOPER`
Once the feature is working without errors we can open it to the correct target audience

Reply:
Totally agree, I will update the PR regarding this point